docs/plans/2026-04-09-phase-7-5-runtime-unification.md

```
@@ -0,0 +1,1124 @@
+        key: String,
+    },
+```
+
+**Step 3: Run check**
+
+Run: `cargo check -p nebula-runtime`
+Expected: compiles.
+
+**Step 4: Commit**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)